### DISCUSSION TITLE
ventura support

### DISCUSSION THREAD

**Discussion by DarthMDev** [The-Wineskin-Project/WineskinServer#293] [General]
Any plans for getting ventura support for this software?
The problem im getting is after creating a blank wrapper from the newest engines it stalls and freezes (tried waiting 10 minute+)

**Comment by emendelson** [comment]
I just created a new wrapper on an M1 Mac running the latest Ventura beta, and it worked perfectly.

If you're having problems, then it may be with the beta version of macOS. Each year's beta versions tend to have problems with this kind of complex, non-standard software. Best thing to do is file a detailed bug report with Apple. When I did this with a Wineskin-related bug a few years ago, Apple fixed it before the final release.

**Comment by Gcenx** [comment]
Considering I know someone who’s been running the dev beta since it landed and it’s working for them I’m assuming this one’s user error.

**Comment by DarthMDev** [comment]
Any idea how I can resolve this, it was working when I was on macOS Monterey.
But maybe it's just a coincidence I guess

**Comment by emendelson** [comment]
Maybe create a new wrapper? Your old one may have got corrupted somehow. Wineskin Winery continues to work perfectly here under Ventura.

**Reply by DarthMDev** [reply]
Nah this is when creating a new wrapper . I did find how to get the useful info log and im sending it to gcenx via discord
